### PR TITLE
docs(readme): add ci, release, license and go report badges

### PR DESCRIPTION
**Pull request by R3DRUN3**
**- Description for the changelog**

*SAFE TO MERGE PR*
just added ci, release, license and go report badges to readme:
[ci](https://github.com/moby/moby/actions/workflows/ci.yml) Latest release License[Go Report Card](https://goreportcard.com/report/github.com/moby/moby)

### CODE REVIEW

**Review by thaJeztah — APPROVED** [review]
LGTM

/cc @crazy-max

File: README.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
 The Moby Project
 ================
+[![ci](https://github.com/moby/moby/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/moby/moby/actions/workflows/ci.yml)
+![Latest release](https://img.shields.io/github/v/release/moby/moby)
+![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)
```
Comment (crazy-max) [inline README.md:5]: Let's add links so users are not confused when clicking on it:

```suggestion
[![Latest release](https://img.shields.io/github/v/release/moby/moby)](https://github.com/moby/moby/releases/latest)
[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://github.com/moby/moby/blob/master/LICENSE)
```

File: README.md [hunk 2]
```diff
@@ -1,8 +1,14 @@
 The Moby Project
 ================
+[![ci](https://github.com/moby/moby/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/moby/moby/actions/workflows/ci.yml)
```
Comment (crazy-max) [inline README.md:3]: I would put CI related badges after `Latest release` and `License`. This is not a primary info imo. We should also add the test workflow:

```suggestion
[![ci workflow](https://github.com/moby/moby/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/moby/moby/actions/workflows/ci.yml)
[![test workflow](https://github.com/moby/moby/actions/workflows/test.yml/badge.svg?branch=master)](https://github.com/moby/moby/actions/workflows/test.yml)
```

File: README.md [hunk 3]
```diff
@@ -1,8 +1,14 @@
...
+![Latest release](https://img.shields.io/github/v/release/moby/moby)
+![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)
+[![Go Report Card](https://goreportcard.com/badge/github.com/moby/moby)](https://goreportcard.com/report/github.com/moby/moby)
 
 ![Moby Project logo](docs/static_files/moby-project-logo.png "The Moby Project")
 
+
+
```
Comment (crazy-max) [inline README.md:11]: ?

**Pull request closed without merging**